### DISCUSSION TITLE
I want to get followers and starts

### DISCUSSION THREAD

**Discussion by superdev223** [community/community#58111] [Accessibility]
### Select Topic Area

Question

### Body

I want to get followers and repository starts as much as possible. How can I get?

**Comment by HappyMicky0317** [comment]
To get more followers and repository stars on GitHub, you can try the following strategies:

1. Create high-quality repositories: Make sure your repositories contain well-documented, useful, and unique code. This will make it more likely that people will star your repositories.

2. Engage with the community: Participate in forums, answer questions, and contribute to open source projects. This will help you build relationships with other developers and increase your visibility.

3. Use social media: Share your work on social media platforms like Twitter, LinkedIn, and Reddit. Use relevant hashtags and engage with other users to increase your visibility.

4. Attend events: Attend conferences, meetups, and hackathons to network with other developers and showcase your work.

5. Collaborate with others: Collaborating with other developers on open source projects can help you build your reputation and attract more followers.

6. Optimize your profile: Make sure your profile is complete, including a profile picture and bio. Use keywords in your bio to make it easier for others to find you.

Remember that building a following takes time and effort, so be patient and persistent in your efforts.

**Reply by superdev223** [reply]
Thank you for your kind reply.

**Comment by ghostinhershell** [comment]
@superdev223  & @HappyDream0317, 

GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
